### DISCUSSION TITLE
mac-mouse-fix vs mos?

### DISCUSSION THREAD

**Discussion by seagle0128** [noah-nuebling/mac-mouse-fix#116] [Questions] [answered]
Hi , could you please clarify what's the difference between `mac-mouse-fix` and [Mos](https://github.com/Caldis/Mos)? Thanks!

**Comment by noah-nuebling** [comment, marked as answer]
Hey there Vincent, thanks for reaching out!

The main difference is that Mac Mouse Fix has button-remapping features, too, while MOS is solely focused on scrolling.

Mac Mouse Fix  also has simpler options for the scrolling.

**Reply by seagle0128** [reply]
Thank you!

**Reply by noah-nuebling** [reply]
Sure thing :)
